Changes

```diff
@@ -362,6 +362,10 @@ OCaml 4.13.0
 - #10232: Warning for unused record fields.
   (Leo White, review by Florian Angeletti)
 
+- #10407: Produce more detailed error messages that contain full error
+  #traces when module inclusion fails.
+  (Antal Spector-Zabusky, review by Florian Angeletti)
```

Comment:
Note that the change entry needs to be moved in the "Working version" section.

Reply:
I got sniped by the move to 4.13 :-)